tests/test_type_adapter.py

```diff
@@ -2,13 +2,14 @@
 import sys
 from dataclasses import dataclass
 from datetime import date, datetime
-from typing import Any, Dict, ForwardRef, Generic, List, NamedTuple, Tuple, TypeVar, Union
+from typing import Any, Dict, ForwardRef, Generic, List, NamedTuple, Optional, Tuple, TypeVar, Union
 
 import pytest
 from pydantic_core import ValidationError
-from typing_extensions import Annotated, TypeAlias, TypedDict
+from typing_extensions import Annotated, Literal, TypeAlias, TypedDict
 
-from pydantic import BaseModel, TypeAdapter, ValidationInfo, field_validator
+import pydantic
```

Comment:
Let's just import pydantic's `dataclass` directly instead of doing this catchall import

Reply:
Done